config/crd/bases/appstudio.redhat.com_components.yaml

```diff
@@ -333,6 +333,10 @@ spec:
                       component
                     type: string
                 type: object
+              pipeline:
```

Comment:
How would this be filled?

Reply:
Name of pipeline to use for build, e.g. `devfile-build`, `docker-build`, etc.